src/main/java/ninjabrainbot/io/KeyPresser.java

```diff
@@ -28,15 +28,13 @@ public void paste(String s) throws InterruptedException {
 		StringSelection selection = new StringSelection(s);
 		clipboard.setContents(selection, null);
 		robot.keyPress(CTRL_KEYCODE);
-		Thread.sleep(50);
 		robot.keyPress(KeyEvent.VK_V);
+		Thread.sleep(60);
 		robot.keyRelease(CTRL_KEYCODE);
-		Thread.sleep(50);
 		robot.keyRelease(KeyEvent.VK_V);
 	}
 
-	public void releaseF3C() {
```

Comment:
Why change this from releasing F3 to C?

Reply:
Normally, F3+C doesn't toggle the debug menu. But if F3 is released twice (once by `robot.keyRelease(KeyEvent.VK_F3)` and again by manually releasing the key), then it does. While not a huge deal, this can be rather annoying. Of course correct me if I'm wrong, but I don't think having F3 pressed can mess with anything.

C was already being released in this method, and from testing it seems like chat sometimes doesn't open without `robot.keyRelease(KeyEvent.VK_C)`.

The main change I made relating to this method is when it get called. It was previously only being called for the first F3+C, but now gets called for each F3+C. I did this to fix the issue mentioned above about chat sometimes not opening. If I didn't remove the `robot.keyRelease(KeyEvent.VK_F3)`, then this change would make the debug menu toggling issue much more annoying.